main_Function_Simulation.py

```diff
@@ -0,0 +1,14 @@
+def main_Function_Simulation(user_input, u):
+
+    c = def_Courant_Number(user_input['Time Steps'],user_input['Space Steps'],user_input['Wave Speed'])
+    
+    for j in range(2,user_input['Temporal Nodes']):
+        for i in range(1,user_input['Spatial Nodes']):
```

Comment:
A matrix-vector representation of the spatial update makes this a little easier to follow
```
u[j,:] = A * u[j-1,:] - u[j-2,:] + f[j,:]
```